Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR.azsli

```diff
@@ -5,6 +5,9 @@
  * SPDX-License-Identifier: Apache-2.0 OR MIT
  *
  */
+
+#pragma once
+#define STANDARDMULTILAYERPBR_MATERIAL_TYPE 1
```

Comment:
Do you need this specific define here? As far as i can tell, nothing in the entire codebase reacts to it, and if you are building a custom material based on this one, you can set that define when you include this file...